Blish HUD/_Utils/WindowUtil.cs

```diff
@@ -223,6 +224,10 @@ public enum OverlayUpdateResponse {
                 DwmExtendFrameIntoClientArea(winHandle, ref marg);
             }
 
+            if (!wasOnTop) {
+                SetLayeredWindowAttributes(winHandle, 0, 255, LWA_ALPHA);
```

Comment:
If not hiding the window, I might need a better place for this then, a place where it executes only once when the window is ready to be seen.